src/xcode/ENA/ENA/Source/Extensions/UIViewController+Alert.swift

```diff
@@ -18,10 +18,13 @@
 import Foundation
 import UIKit
 extension UIViewController {
-	func alertError(message: String?, title: String?, completion: (() -> Void)? = nil) {
+	func alertError(message: String?, title: String?, optinActions: [UIAlertAction]? = nil, completion: (() -> Void)? = nil) {
```

Comment:
if it's camelcase, shouldn't it be optInActions?

Reply:
done